README.md

```diff
@@ -114,8 +115,32 @@ context.EndRequest += (sender, args) =>
 };
 ```
 
-Client trace example
+#### Client trace example
 Trace before and after a remote request call
+
+```
+var zipkinClient = (ITracerClient)HttpContext.Current.Items["zipkinClient"];
+var url = "https://abc.xyz.com:8000";
+var requestUri = "/object/1";
+HttpResponseMessage result;
+using (var client = new HttpClient())
+{
+    client.BaseAddress = new Uri(url);
+
+	// start client trace
+    var span = tracerClient.StartClientTrace(new Uri(client.BaseAddress, requestUri), "GET");
+
+    result = await client.GetAsync(requestUri);
+
+	// end client trace
+    tracerClient.EndClientTrace(span);	
+}
+...
+```
+
+#### Recording arbitrary events and additional information
+Additional annotations can be recorded by using the ZipkinClient's `Record()` and `RecordBinary&lt;T&gt;()` methods:
```

Comment:
Thanks, I was not sure how to escape the <> signs, but it seems that Github will ignore it as a HTML and render it correctly.